web_widget_remaining_days_exact_date/tests/test_main.py

```diff
@@ -0,0 +1,110 @@
+from odoo import exceptions
+from odoo.tests import common, tagged
+
+
+@tagged("post_install", "-at_install")
+class TestWidgetRemainingDaysExactDate(common.TransactionCase):
```

Comment:
Please inherit from [BaseCommon](https://github.com/odoo/odoo/blob/9fc9559ef06908eb788c491132d3aa1a1de91b8f/odoo/addons/base/tests/common.py#L20)
```suggestion
class TestWidgetRemainingDaysExactDate(BaseCommon):
```